test/test_dispatch.py

```diff
@@ -532,21 +532,21 @@ def test_computed_table_with_ambiguous_autogradother(self):
             lambda m: m.def_("foo(Tensor x) -> Tensor"),
             # m.impl("foo", torch::kCompositeImplicitAutograd, [](const Tensor & x) { return x })
             lambda m: m.impl_t_t("foo", "CompositeImplicitAutograd", debug="fn_math"),
-            # m.impl("foo", torch::kQuantizedCPU, [](const Tensor & x) { return x })
-            lambda m: m.impl_t_t("foo", "QuantizedCPU", debug="fn_quantizedcpu"),
+            # m.impl("foo", torch::kFPGA, [](const Tensor & x) { return x })
+            lambda m: m.impl_t_t("foo", "FPGA", debug="fn_fpga"),
```

Comment:
I had to remove `QuantizedCPU` from these tests for the same reason as https://github.com/pytorch/pytorch/pull/69633/files#r774748677